kedro/framework/cli/starters.py

```diff
@@ -28,7 +33,7 @@
 KEDRO_PATH = Path(kedro.__file__).parent
 TEMPLATE_PATH = KEDRO_PATH / "templates" / "project"
 
-_STARTER_ALIASES = {
+_OFFICIAL_STARTER_ALIASES = {
```

Comment:
Is this still used anywhere? I feel like it just duplicates `_OFFICIAL_STARTERS_SPECS` so could be removed?